fuel-relayer/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "fuel-relayer"
+version = "0.1.0"
```

Comment:
Set version number to the latest release, unless we don't want to publish this yet, at this point set to `0.0.0` and `publish = false`.